File: DBtoCSharp_MARIADB/output/INNODB_LOCK_WAITS.ts
export interface INNODB_LOCK_WAITS
    {
        /**///  */
        requesting_trx_id: number;
        /**///  */
        requested_lock_id: string;
        /**///  */
        blocking_trx_id: number;
        /**///  */
        blocking_lock_id: string;
}

File: DBtoCSharp_MARIADB/modelos/INNODB_LOCK_WAITS.cs

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    public class INNODB_LOCK_WAITS
    {
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long requesting_trx_id { get; set; }
            [Required]
    [StringLength(81)]
        /// <summary>
        /// 
        /// </summary>
        public string requested_lock_id { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long blocking_trx_id { get; set; }
            [Required]
    [StringLength(81)]
        /// <summary>
        /// 
        /// </summary>
        public string blocking_lock_id { get; set; }

}
}
